test/verifiedTests/reconstruction/testModelManipulation/testDynamicModelFieldModification.m

```diff
@@ -0,0 +1,170 @@
+% The COBRAToolbox: testDynamicModelFieldModification.m
+%
+% Purpose:
+%     - Test functionality in removeFieldEntriesForType, extendModelFields,
+%     getModelFieldsForType
+%
+% Author:
+%     - Original file: Thomas Pfau
+
+global CBTDIR
+
+% save the current path
+currentDir = pwd;
+
+% initialize the test
+fileDir = fileparts(which('testDynamicModelFieldModification.m'));
+cd(fileDir);
+
+%Lets create an empty model.
+model = createModel();
+
+%Lets add a gene:
+model.genes{end+1,1} = 'gene1';
+
+%extend all relevant fields (the original size was 0.
+%get all Model fields for genes
+fprintf('Testing getModelFieldsForType and extendModelFielsForType ...\n');
+[matchingGeneFields,dimensions] = getModelFieldsForType(model,'genes','fieldSize',0);
+assert(dimensions == 2);
+assert(isequal(matchingGeneFields,{'rxnGeneMat'}));
+model = extendModelFieldsForType(model,'genes');
+%This should result in an empty rxnGEneMatrix of size 0x1
+assert(all(size(model.rxnGeneMat) == [0,1]));
+
+%Recreate the model.
+model = createModel();
+
+%lets add a couple of reactions
+Reactions = {'R1',{'A','B'},[-1 1],0,1000;...
```

Comment:
May you rename the variable here?

Reply:
Why?

Reply:
... as we are trying to stick to `camelToe` format ...`reactions` 😄